elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexTransaction.java

```diff
@@ -150,43 +139,6 @@ protected DataStoreTransaction getTransaction(Class<?> cls) {
     }
 
     @Override
-    public <T> Object getRelation(
```

Comment:
Yeah, good catch. I must have been getting tired around here. This interface must never have been updated with the new code (this `getRelation` is the Elide 2.0 interface).